packages/icons/src/library/navigation-menu.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * WordPress dependencies
+ */
+import { SVG, Path } from '@wordpress/primitives';
+
+const navigationMenu = (
```

Comment:
This one seems to be named "block-navigation" in figma:

![image](https://user-images.githubusercontent.com/548849/106611632-7e9aa400-6568-11eb-9f01-136065a0f700.png)

@jasmussen can you check what should be the desired name?

Reply:
Nice find!
I'll wait for a confirmation and update the icon name.